src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -57,6 +58,8 @@ public void periodic() {
         Units.degreesToRadians(result.getBestTarget().getPitch()))
         );
       targetAngle = result.getBestTarget().getYaw();
+      SmartDashboard.putNumber("distance", targetDistance);
```

Comment:
do we want to keep these in main code? I don't think operators are going to much debugging during a match. we should try to keep shuffleboard kinda organized imo

Reply:
We can clean up before guilford